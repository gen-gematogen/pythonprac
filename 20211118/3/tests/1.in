alp = Alpha(m=11, n=21, a=42)
alp.g = "HI!"
print(alp)
alq = AlphaQ(m=11, z=21, a=42)
alq.g = "HI!"
print(alq)
