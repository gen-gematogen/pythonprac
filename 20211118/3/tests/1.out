a: 42, g: HI!, m: 11, n: 21
a: 42, g: HI!, m: 11, z: 21
